mosquitto.conf

```diff
@@ -0,0 +1,5 @@
+listener 1883
+persistence false
```

Comment:
this will disable buffering messages correct?

Reply:
Yes, if we want to enable it later we can, but for now I don't want to fill up our sd card.